lib/tiled/map.rb

```diff
@@ -89,6 +89,28 @@ def pixelheight
       return height * tileheight
     end
 
+    #gets the first object layer with the passed name of `layer_name`
+    #then looks in that layer for the first object that has the name `object_name`
+    #then returns that object
+    def object_by_name(layer_name, object_name)
```

Comment:
I'm not sure about passing a layer name there, because this action can be performed in this way:
```ruby
map.layers[layer_name].find_by_name(object_name)
```

But it would be more valuable if the user will be able to find an object without passing `layer_name`:
```ruby
map.object_by_name(object_name)
```

What do you think about this?